### PR TITLE
Feature/main

### PR DESCRIPTION
**Pull request by MahlekenyaneTseole**

### CODE REVIEW

File: chatterbox/src/App.js [hunk 1]
```diff
@@ -22,7 +22,6 @@ class App extends Component {
         this.funcVar = setInterval(this.splash, 2000);
     }
     splash() {
-        history.push('/auth/login');
         this.setState({ loading: false });
```
Comment (KimberleighBeyl) [inline chatterbox/src/App.js]: this.props.loading

**Review by KimberleighBeyl — CHANGES_REQUESTED** [review]
Change to this.props.loading and add to reducer

**Comment by CeeyarRbt** [comment]
Resolved conflicts

File: chatterbox/src/App.js (hunk 1 above)
Comment (CeeyarRbt) [reply chatterbox/src/App.js]: done.

**Review by KimberleighBeyl — APPROVED** [review]
LGTM

**Pull request merged**